tasks/test.py

```diff
@@ -153,6 +153,30 @@ def lint_teamassignment(ctx):
     else:
         print("PR not yet created, skipping check for team assignment")
 
+@task
+def lint_milestone(ctx):
+    """
+    Make sure PRs are assigned a milestone
+    """
+    pr_url = os.environ.get("CIRCLE_PULL_REQUEST")
+    if pr_url:
+        import requests
+        pr_id = pr_url.rsplit('/')[-1]
+
+        # first check 'noreno' label
```

Comment:
this comment seems irrelevant here:

```suggestion
```

Reply:
I added a line in the contributing page.

For your second point your're right and that's already what we do. But before creating test card having a list of the needed PR for a release is very useful. Once we no that everything we want if merge, we can run the script and create cards automatically.